cypress/integration/tests/administrator/repository/maven.test.ts

```diff
@@ -115,6 +113,11 @@ describe("Test secure and insecure maven repository analysis", () => {
         application.openreport();
     });
 
+    // The following test case should verify if the analysis is failed rather than completed,
+    // but due to how maven structure works it will always pick the ( http ) link as ( https )
+    // which means, it will always pass regardless if the insecure toggle is disabled or not
+    // todo: find a workaround to this
+
```

Comment:
@midays PR LGTM :+1: 
But the commented lines are part of or related to which test can be confusing.
I think we should mention the ToDo inside the test.

Reply:
Changed location to inside the TC

Reply:
Great, Thanks @midays :+1: